sklearn/linear_model/randomized_l1.py

```diff
@@ -307,7 +311,7 @@ class RandomizedLasso(BaseRandomizedLinearModel):
 
     See also
     --------
-    RandomizedLogisticRegression, LogisticRegression
+    RandomizedLogisticRegression, Lasso, ElasticNet
```

Comment:
I don't think it would hurt to mention `LogisticRegression` here. Though yes, you could argue that `LogisticRegression` serves a different purpose than a feature selection method, it might help users distinguish the use cases of these methods.